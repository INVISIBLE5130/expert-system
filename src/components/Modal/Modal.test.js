const renderer = require('react-test-renderer')
const {Modal} = require('./Modal')

it("Check modal", () => {
    const ModalComponent = renderer.create(Modal).toJSON();
    expect(ModalComponent).toMatchSnapshot();
});