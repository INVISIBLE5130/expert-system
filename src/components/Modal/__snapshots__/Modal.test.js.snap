// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check modal 1`] = `null`;
